index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="description" content="Personal Portfolio of Susana Sanchez" />
     <meta name="keywords" content="Portfolio, About Me, Experience, Contact" />
     <meta name="author" content="Susana Sanchez" />
-    <title>Lesson 7</title>
+    <title>Lesson 8</title>
     <style>
```

Comment:
since your css file is now controlling styling, its good practice to remove this styling that you have coded inline in the html file :). Notice that there arent any visible changes to your page when you remove it - thats because your index.css file that you have linked to is overriding these styling changes. You can remove the entire style block and all contents between <style> and </style>